dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/localfiletreetable/Ds3GetJob.java

```diff
@@ -89,6 +87,7 @@ public void setJobID(final UUID jobID) {
 
     @Override
     public void executeJob() throws Exception {
+        final Calendar jobStartTime = Calendar.getInstance();
```

Comment:
The new java 8 time libraries should be used rather than the old pre java 8 time libraries.